pkg/pc/fields/fields.go

```diff
@@ -53,6 +54,12 @@ type PodCluster struct {
 	// Selector to identify the pods that are members of this pod cluster
 	PodSelector labels.Selector
 
+	// AllocationStrategy tweaks certain characteristic about how pods
+	// within this cluster are managed. For example the "pet" strategy will
+	// never transfer a pod from one node to another without human
+	// intervention whereas the "cattle" strategy will
+	AllocationStrategy rc.Strategy `json:"allocation_strategy"`
```

Comment:
`RawPodCluster` contains the json encoding/decoding stuff for the other fields. Is it an intentional choice to include that here?

Reply:
IDK what i was thinking, I missed a lot of stuff. should be fixed now!

Reply:
actually i'm adding to the tests too